packages-exp/auth-exp/src/core/strategies/email_and_password.ts

```diff
@@ -82,3 +84,14 @@ export async function verifyPasswordResetCode(
   // Email should always be present since a code was sent to it
   return data.email!;
 }
+
+export async function signInWithEmailAndPassword(
```

Comment:
This method need not be declared `async` since you're not awaiting anything

Reply:
signInWithCredential is async

Reply:
Right, but all `async` does is allow you to use `await` syntax rather than promise chaining. If you don't use `await` in the body of the function, you don't need `async`. Declaring something `async` also generates boilerplate in the compiled code so if you're not awaiting anything it's best to avoid it.